app-shared/src/iosMain/kotlin/io/github/droidkaigi/confsched/navigation/graph/SessionsNavGraph.kt

```diff
@@ -89,7 +89,9 @@ private fun NavGraphBuilder.searchNavGraph(
     onTimetableItemClick: (TimetableItemId) -> Unit,
 ) {
     composable<SearchRoute> {
-        with(rememberSearchScreenContextRetained()) {
+        // Only on the JVM is it necessary to specify the `rememberRetained` key.
+        // This is not required on iOS.
+        with(rememberSearchScreenContextRetained(0L)) {
```

Comment:
When we passed arguments to createSearchScreenContext, we had no choice but to pass values in this manner even outside the JVM. 🥹